content/v2.0/actions/overview.md

```diff
@@ -62,8 +62,6 @@ Topics to cover:
 - request and response
 - mime types and format
 - control flow
-- params (including req.ar.to_h) and query string parameters.
```

Comment:
What is `req.ar` ? This method doesn't seem to be defined.

Reply:
@swilgosz no such thing, must be some kind of a mistake here